### PR TITLE
Update axios in samples

### PR DESCRIPTION
**Pull request by jasonnutter**

### CODE REVIEW

File: samples/package.json [hunk 1]
```diff
@@ -7,11 +7,11 @@
         "test": "jest"
     },
     "dependencies": {
-        "axios": "^0.21.1",
+        "axios": "^0.21.4",
```
Comment (jasonnutter) [inline samples/package.json:10]: @tnorling Is the package-lock needed for this, or can we remove it like we do for other samples?

Comment (sameerag) [reply samples/package.json:10]: Is this change made only for node in the last build?

Comment (tnorling) [reply samples/package.json:10]: We use it for dependency caching in the e2e workflows. If we want to remove it we'd have to look into using something else as the cache key

Comment (jasonnutter) [reply samples/package.json:10]: > Is this change made only for node in the last build?

Not sure what you mean exactly, but msal-node is already on teh right version.

> We use it for dependency caching in the e2e workflows. If we want to remove it we'd have to look into using something else as the cache key

Got it, will keep.

Comment (sameerag) [reply samples/package.json:10]: Yeah, I remember making this change for msal-node. I did not realize the samples also use `axios`

**Review by sameerag — APPROVED** [review]

**Review by tnorling — APPROVED** [review]

**Pull request merged**